community/osinfo-db-tools/APKBUILD

```diff
@@ -1,6 +1,6 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=osinfo-db-tools
-pkgver=1.1.0
+pkgver=1.2.0
```

Comment:
Makes sense to upgrade `libosinfo` to 1.2.0 in the commit set https://releases.pagure.org/libosinfo/